save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/docker/DockerContainerManager.kt

```diff
@@ -1,112 +0,0 @@
-package com.saveourtool.save.orchestrator.docker
-
-import com.saveourtool.save.domain.Sdk
-import com.saveourtool.save.orchestrator.config.ConfigProperties
-import com.saveourtool.save.orchestrator.execTimed
-import com.saveourtool.save.orchestrator.getHostIp
-
-import com.github.dockerjava.api.DockerClient
-import com.github.dockerjava.api.command.BuildImageResultCallback
-import com.github.dockerjava.api.model.Image
-import io.micrometer.core.instrument.MeterRegistry
-import org.slf4j.LoggerFactory
-import org.springframework.stereotype.Component
-
-import java.io.File
-
-import kotlin.io.path.createTempDirectory
-import kotlin.io.path.createTempFile
-
-/**
- * A class that communicates with docker daemon
- */
-@Component
-@Suppress("AVOID_NULL_CHECKS")
-class DockerContainerManager(
-    private val configProperties: ConfigProperties,
-    private val meterRegistry: MeterRegistry,
-    private val dockerClient: DockerClient,
-) {
-    /**
-     * Creates a docker image with provided [resources]
-     *
-     * @param baseImage base docker image from which this image will be built
-     * @param imageName name which will be assigned to the image
-     * @param runCmd `RUN` directives to be added to Dockerfile *before* resources from `baseDir` are copied (so that resources-agnostic command
-     * results can be cached in docker layers).
-     * @return id of the created docker image
-     * @throws DockerException
-     */
-    @Suppress(
-        "TOO_LONG_FUNCTION",
-        "LongMethod",
-        "LongParameterList",
-        "TOO_MANY_PARAMETERS"
-    )
-    internal fun buildImage(baseImage: String = Sdk.Default.toString(),
-                            imageName: String,
-                            runCmd: String = "RUN /bin/bash",
-    ): String {
-        val tmpDir = createTempDirectory().toFile()
-        val dockerFile = createDockerFile(tmpDir, baseImage, runCmd)
-        val hostIp = getHostIp()
```

Comment:
https://github.com/saveourtool/save-cloud/issues/728 could be closed?

Reply:
No, host IP is still required for containers and this mapping is still being added when creating containers